### PR TITLE
cohort 4: [answering questions] add recommend to talk to peers

### PR DESCRIPTION
**Pull request by jonleung**

### CODE REVIEW

File: cohort_4/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -919,6 +919,10 @@ on Google (ex. if they don't know about the existence of the JavaScript
 console and it's the tool they need), then feel free to explain it to them
 directly. Use your discretion for when to do this.
 
+Moreover, if you know that someone else has solved the problem or think it is
+likely that another hacker may have the solution to this, you could recommend
+that the hacker reach out to one of their peers.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: The perspective of the writing changes from first person to second person in this paragraph. Can you fix that?

Comment (jonleung) [inline cohort_4/playbook/ACTIVITIES.md]: @zachlatta added a fix

File: cohort_4/playbook/ACTIVITIES.md [hunk 2]
```diff
@@ -919,6 +919,10 @@ on Google (ex. if they don't know about the existence of the JavaScript
 console and it's the tool they need), then feel free to explain it to them
 directly. Use your discretion for when to do this.
 
+Moreover, we recommend that if you know that someone else has solved the problem
+or think it is likely that another hacker may have the solution to this, you
+could recommend that the hacker reach out to one of their peers.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: I think the usage of "someone else" in this section is pretty ambiguous. "someone else" as in another hacker? Or another organizer?

Comment (jonleung) [inline cohort_4/playbook/ACTIVITIES.md]: Will work on this by end of day tomorrow @zachlatta

Comment (jonleung) [inline cohort_4/playbook/ACTIVITIES.md]: @zachlatta just pushed a fix

File: cohort_4/playbook/ACTIVITIES.md [hunk 3]
```diff
@@ -919,6 +919,10 @@ on Google (ex. if they don't know about the existence of the JavaScript
 console and it's the tool they need), then feel free to explain it to them
 directly. Use your discretion for when to do this.
 
+Moreover, we recommend that if you know that a hacker has solved the problem
+or think it is likely that another hacker may have the solution to this, you
+could recommend that the hacker reach out to one of their peers.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Why? Should we add the reasoning behind this to this section?

Also, this wording feels pretty indirect.

> "We recommend...[that] you could recommend that the hacker reach out..."

What if we just said "We recommend...that you encourage hackers to reach out to their peers in this situation"?

Comment (jonleung) [inline cohort_4/playbook/ACTIVITIES.md]: Fixed

**Comment by jonleung** [comment]
@zachlatta made the edit

File: cohort_4/playbook/ACTIVITIES.md [hunk 4]
```diff
@@ -919,6 +919,11 @@ on Google (ex. if they don't know about the existence of the JavaScript
 console and it's the tool they need), then feel free to explain it to them
 directly. Use your discretion for when to do this.
 
+Moreover, we recommend that you  encourage hackers to reach out to their peers
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md:922]: There are two spaces where there should only be one between "you" and "encourage".

**Comment by zachlatta** [comment]
Cherry picked into master with minor phrasing and grammar changes in ad8e2f4f554ea39ac451dc40305c464271d7bf9f.

**Pull request closed without merging**